packages/teams-js/src/internal/appHelpers.ts

```diff
@@ -33,6 +40,16 @@ const initializationTimeoutInMs = 5000;
 
 const appLogger = getLogger('app');
 
+/**
+ * The response of the notify success callback.
+ */
+export interface NotifySuccessResponse {
+  /**
+   * It shows if the callback resolved successfully in the host. If the host does not support answering back to the callback, the result is unknown.
```

Comment:
I think Niharika mentioned that the notifySuccess is only to track up to the hub sdk receiving it? Might be good to check with her

Reply:
Of what I understood the idea was to resolve the promise when the Hub finishes processing the message, that is rewriting the function from a fire-and-forget to a request-and-wait-for-response. If this assumption is wrong, please correct me Niharika

Reply:
@maglims Biz Chat requested that need to know if app.notifySuccess has been successfully resolved to help plugin holes in the telemetry. They are trying to get 99.99% success rate for P95 telemetry and because of the missing telemetry from hub/hubsdk, they want to rely on their own telemetry to fund the effort. This might help in moving the 'Other' percentage to 'Success' bucket once the biz chat gets a response back from hubsdk that it indeed received and processed notifysuccess.